be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -117,7 +120,16 @@ Status VOlapTableSink::send(RuntimeState* state, vectorized::Block* input_block)
             continue;
         }
         _partition_ids.emplace(partition->id);
-        uint32_t tablet_index = dist_hash % partition->num_buckets;
+        if (findTabletMode != FindTabletMode::FIND_TABLET_EVERY_ROW) {
```

Comment:
How to deal with find tablet every sink?

Reply:
if we find first tablet for one partition, we don't clear the partition_to_tablet_map before processing every row batch